src/roman_datamodels/stnode.py

```diff
@@ -37,6 +39,20 @@
     "TaggedScalarNode",
 ]
 
+META_REGISTER = True
+
+
+@contextmanager
+def registration_off():
+    """
+    Context manager to turn off registration of classes.
+    """
+
+    global META_REGISTER
+    META_REGISTER = False
+    yield
+    META_REGISTER = True
+
```

Comment:
Enables temporary turning off registration of tagged object subclasses, so that the type stubs can be loaded as a module without breaking anything.